red-controller/src/main/java/com/masi/red/controller/TestUIController.java

```diff
@@ -46,11 +47,12 @@ public String getJobTestPage(@PathVariable Integer jobId, Model model){
     @PostMapping(value = "/kandydat/stanowisko/{jobId}")
     @ResponseBody
     public ResponseEntity<Object> getPostJobTestPage(@PathVariable Integer jobId, @Valid @RequestBody CandidateAnswerDTO answerDto){
-
-
+        int answersNo = answerDto.getAnswers().size();
         System.out.println(answerDto);
-
-        return new ResponseEntity<>(true, HttpStatus.OK);
+        for (int counter = 0; counter <answersNo; counter++) {
+            answerService.addAnswers(answerDto, counter);
```

Comment:
Sprawdź komentarz do metody w serwisie. 

Dodatkowo możesz spróbować użyć tutaj stream'a zamiast zwyklego for loop'a.